Enterprise-CMCS/eAPD
# bbrooks/4031-Update-Packer-Cleanup-20221102

Resolves # 4031

### Description
The Packer cleanup script was outdated and did not cleanup all of the different AMIs that we were creating. This update fixes a bug that cause some in-use AMIs to get deleted and adds support for Mongo AMIs.

### Significant changes or possible side effects

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |

### Steps to manually verify this change

1. Pull down branch 
2. Change deregister from aws deregister command to something like "echo $AMI would be deregistered"
3. Run script and make sure that the most recent and in use AMIs are all labeled as "Keeping ami AMI ID"

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
